src/lib/utils.tsx

```diff
@@ -69,3 +73,139 @@ export const useProjectId = (props: { projectId?: string }) => {
 
   return projectId;
 };
+
+/**
+ * 
+ * @param data
+ * text data
+ * @param count 
+ * the variable number used to determine which plural case to use
+ * zero = 0
+ * one = 1
+ * two = 2
+ * few = 3,4,5
+ * many = 6,7,...,99
+ * other = 100, 101, ...
+ * if count is provided but not matching plural key, fallback to base plural value
+ * in future we should also user's to define their own middleware for picking plurals
+ */
+const getPluralText = (data: any, count: Count) => {
+  if (count === undefined|| Object.keys(data?.plurals || {})?.length === 0) {
+    return data.text
+  }
+  else if (count === 0) {
+    //zero
+    if (data.plurals.zero) return data.plurals.zero
+    return data.text
+  } else if (count === 1) {
+    // one
+    if (data.plurals.one) return data.plurals.one
+    return data.text
+  } else if (count === 2) {
+    // two
+    if (data.plurals.two) return data.plurals.two
+    return data.text
+  } else if (count >= 3 && count <= 5) {
+    // few
+    if (data.plurals.few) return data.plurals.few
+    return data.text
+  } else if (count >= 6 && count <= 99) {
+    // many
+    if (data.plurals.many) return data.plurals.many
+    return data.text
+  } else {
+    // other
+    if (data.plurals.other) return data.plurals.other
+    return data.text
+  }
+}
+
+export const interpolateVariableText = (data: any, variables: Variables, count: Count) => {
```

Comment:
Can we get rid of this `any`? I think it's important for reasons I described [here](https://github.com/dittowords/ditto-react/pull/10/files#r784859961).

Reply:
👍